Documentation/Migration/Index.rst

```diff
@@ -78,6 +90,20 @@ PHP-based rendering tool:
 
 ..  rst-class:: bignums
 
+#.  Manual modifications of the :file:`guides.yml`
+
+    You have to manually change the following: in the extension tag add the attribute
+
+        ..  code-block:: text
+            :caption: Changes in your :file:`guides.yml`
+
+            interlink-shortcode="vendor/ext_name"
+
+    This you can find in your composer.json under "name". E.g. :json:`friendsoftypo3/jumpurl`.
+    We recommend to reformulate the code using the mac short cut `cmd + option/alt + l` and to
```

Comment:
What about Windows/Linux users? No Max shortcut? Additionally, use the :kbd: role.

Additional "reformulare" sounds strange, better "reformat"

Reply:
@brotkrueml Suggest a nice sentence then I commit it.